lab-taylor/data/order/10222e60-1e3e-11e6-b610-674715db42ad

```diff
@@ -0,0 +1 @@
+{"id":"10222e60-1e3e-11e6-b610-674715db42ad","created":1463716258886,"item":"my item","qty":9393}
```

Comment:
In the future you should gitignore your 'data' directory, anything your program needs to run (like the 'data' directory) should be created at runtime.  This will keep your repository a more manageable size and keep others from downloading your data files or uploading their own accidentally.